web/src/containers/ApdExport.js

```diff
@@ -28,9 +29,9 @@ const ExportAndSubmit = ({ push: pushRoute, useParams }) => {
           </p>
           <Button
             size="big"
-            variation="primary"
+            variation={years.length > 0 ? "primary": 'danger'}
```

Comment:
I would suggest instead of showing this as the danger variant make the button disabled so it doesn't signify it can be clicked.

Something like
`disabled={years.length > 0 ? true : false}`

Reply:
fixed in incoming push.  I looked at the code, not the docs and didn't realize disabled was a valid option.